core/src/block.rs

```diff
@@ -870,10 +846,7 @@ impl From<&CommittedBlock> for Vec<Event> {
             .into(),
         );
 
-        tx.chain(rejected_tx)
-            .chain(invalid_blocks)
```

Comment:
Why do we exclude `invalid_blocks`?

Reply:
I removed block invalidation because they serve no purpose until we add partial rolebacks as a handling for softforks. https://github.com/hyperledger/iroha/issues/1170 I have a solution in mind for this, so the invalidated blocks will be added back in later.